components/usage/pkg/server/server.go

```diff
@@ -36,7 +35,8 @@ type Config struct {
 
 	StripeCredentialsFile string `json:"stripeCredentialsFile,omitempty"`
 
-	ContentServiceAddress string `json:"contentServiceAddress,omitempty"`
+	// Deprecated
```

Comment:
Why do we need to deprecate this? Because we're no longer uploading usage reports to object storage?

Reply:
Yep, that's correct. I want to ensure that the service doesn't fail to start with config which does set it in case we rollback the version but not the config.